metadata_tools.py

```diff
@@ -71,35 +62,20 @@ def exif_attach_metadata(self, exif_code: int, exif_value):
         """attaches exif metadata tags to image using ExifTools subprocess in command line
         args:
             path: path to image
-            exif_dict: dictionary of exif terms using exif codes, and new values to assign"""
-
-        exif_tag = self.exif_code_to_tag(exif_code)
-        command = ['exiftool', '-overwrite_original', f"-{exif_tag}={exif_value}", self.path]
-        subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        self.logger.info("exif added succesfully")
-
-
-    # still doesn't work yet, but close
-    # def remove_exif_tags(self, exif_codes):
-    #     """
-    #     Remove specified EXIF tags from an image.
-    #     args:
-    #         path: The path to the image.
-    #         exif_codes: A list of EXIF codes to remove.
-    #     """
-    #     try:
-    #         # Load EXIF
-    #         exif_data = self.read_exif_metadata(path=self.path, convert_tags=False)
-    #         # Iterate over the tags to remove
-    #         for code in exif_codes:
-    #             if code in exif_data:
-    #                 exif_data[code] = None
-    #
-    #         self.attach_exif_metadata(path=self.path, exif_dict=exif_data)
-    #
-    #         print("EXIF tags removed successfully")
-    #     except Exception as e:
-    #         print(f"Error removing EXIF tags: {e}")
+            exif_dict: dictionary of exif terms using exif codes, and new values to assign
+        """
```

Comment:
only will run subprocess if exif dict is not empty, otherwise skips subprocess pipe